### ISSUE TITLE
Nvidia GeForce Now

### ISSUE THREAD

**Issue by anthr76** [bottlesdevs/programs#87]
Add support (if possible) for GeForce now game streaming. I've tried to run the installer but it just errors out and points to https://nvidia.custhelp.com/app/answers/detail/a_id/4223

**Comment by mirkobrombin** [comment]
Isn't Nvidia GeForce Now working in a browser?

**Comment by anthr76** [comment]
Yes, only for 1080p@60Hz. If you want to do higher resolutions or refresh rates you need the windows or mac client.

**Comment by mirkobrombin** [comment]
oh ok

**Comment by mouse-man** [comment]
Noting that I am from Australia, and our GeForce Now provider doesn't have the 30 series equivalent GPUs yet, so the best we can do is 1080p@60Hz anyway, but there are unofficial electron clients in Snap, Flatpak, and AUR (from what I can see in Manjaro anyway) that should work as if it is the native client, and how I use it.
![2022-07-01_10-25-GFN](https://user-images.githubusercontent.com/2426485/176799053-171dec62-9b3e-4395-9a57-0497d1f10987.png)
Screenshot from the Manjaro Software Installer.
Maybe this is something to try and see if you can get the higher resolution.

**Comment by anthr76** [comment]
Sadly I have explored this and using a 144Hz 1440p monitor nets really poor results :/
